Kitodo/src/main/java/org/goobi/mq/MapMessageObjectReader.java

```diff
@@ -42,127 +42,113 @@
 	private MapMessage ticket;
 
 	/**
-	 * This instantiates a new MapMessageObjectReader which is attached to a
-	 * given MapMessage.
-	 * 
-	 * @param message
+	 * This instantiates a new MapMessageObjectReader which is attached to a given MapMessage.
+	 *
+	 * @param message add description
 	 */
 	public MapMessageObjectReader(MapMessage message) {
-		if (message == null)
+		if (message == null) {
 			throw new IllegalArgumentException("MapMessageObjectReader: null argument in constructor.");
+		}
 		this.ticket = message;
 	}
 
 	/**
-	 * The function getMandatorySetOfString() fetches a Set<String> from a MapMessage.
-	 * This is a strict implementation that requires the collection not to be
-	 * empty and not to contain the null element.
-	 * 
-	 * Please note that the Set is allowed to contain the empty String (“”).
-	 * 
-	 * @param key
-	 *            the name of the set to return
+	 * The function getMandatorySetOfString() fetches a Set&lt;String&gt; from a MapMessage. This is a strict
+	 * implementation that requires the collection not to be empty and not to contain the null element.
+	 *
+	 * <p>Please note that the Set is allowed to contain the empty String (“”).</p>
+	 *
+	 * @param key the name of the set to return
 	 * @return the set requested
-	 * @throws IllegalArgumentException
-	 *             in case that getObject returns null, the returned object is
-	 *             not of type Collection, any of the content elements are not
-	 *             of type String or the collection is empty at all.
-	 * @throws JMSException
-	 *             can be thrown by MapMessage.getObject(String)
+	 * @throws IllegalArgumentException in case that getObject returns null, the returned object is not of type
+	 * 									Collection, any of the content elements are not of type String or
```

Comment:
Here tabs are not used to indent the code, but in a comment. The result looks horrible.